src/ssl.c

```diff
@@ -16623,6 +16623,90 @@ int wolfSSL_CTX_set_max_proto_version(WOLFSSL_CTX* ctx, int ver)
     return sanityCheckProtoVersion(ctx);
 }
 
+static int get_default_min_proto_version()
+{
+    int ret;
+
+    switch (WOLFSSL_MIN_DOWNGRADE) {
+#ifndef NO_OLD_TLS
+    #ifdef WOLFSSL_ALLOW_SSLV3
+        case SSLv3_MINOR:
+            ret = SSL3_VERSION;
+            break;
+    #endif
+    #ifdef WOLFSSL_ALLOW_TLSV10
+        case TLSv1_MINOR:
+            ret = TLS1_VERSION;
+            break;
+    #endif
+        case TLSv1_1_MINOR:
+            ret = TLS1_1_VERSION;
+            break;
+#endif
+#ifndef WOLFSSL_NO_TLS12
+        case TLSv1_2_MINOR:
+            ret = TLS1_2_VERSION;
+            break;
+#endif
+#ifdef WOLFSSL_TLS13
+        case TLSv1_3_MINOR:
+            ret = TLS1_3_VERSION;
+            break;
+#endif
+        default:
+            ret = 0;
```

Comment:
"The getter functions return the configured version or 0 for auto-configuration of lowest or highest protocol, respectively." (https://www.openssl.org/docs/man1.1.1/man3/SSL_get_min_proto_version.html)